### PR TITLE
Adds petcollars to the cmo, captain, and bartender's locker

### PR DESCRIPTION
**Pull request by jughu**
To well yknow give their pet

🆑 Improvedname
tweak: Cmo, captain, and the bartender now get pet collars in their lockers.
/🆑

### CODE REVIEW

**Comment by lzimann** [comment]
i don't think it works on monkey, does it?

**Comment by Mercenaryblue** [comment]
Runtime, hell yes.

Pun Pun? He could do without.

**Comment by jughu** [comment]
monkeys have a neckslot so i assume it works

**Comment by Mercenaryblue** [comment]
Probably. He also start with a top hat nearby. I think it balance out.

**Comment by jughu** [comment]
FUCK I FORGOT THE MOST IMPORANT PART

**Comment by jughu** [comment]
new title new update

**Comment by Cyberboss** [comment]
It does work on monkeys i think...

File: code/game/objects/structures/crates_lockers/closets/secure/security.dm [hunk 1]
```diff
@@ -11,6 +11,7 @@
 	else
 		new /obj/item/storage/backpack/satchel/cap(src)
 	new /obj/item/clothing/neck/cloak/cap(src)
+	new /obj/item/clothing/neck/petcollar(src)
```
Comment (Iamgoofball) [inline code/game/objects/structures/crates_lockers/closets/secure/security.dm]: captain doesn't have a pet on all maps

Comment (Pascal125) [reply code/game/objects/structures/crates_lockers/closets/secure/security.dm]: Renault you literal rarely plays.

Comment (Iamgoofball) [reply code/game/objects/structures/crates_lockers/closets/secure/security.dm]: not on all maps you literal furry

Comment (Pascal125) [reply code/game/objects/structures/crates_lockers/closets/secure/security.dm]: Well then maybe you should get better grammar and suck it up, you literal deuce.

Comment (jughu) [reply code/game/objects/structures/crates_lockers/closets/secure/security.dm]: renault got added to box to and the most main maps is box/meta

Comment (Iamgoofball) [reply code/game/objects/structures/crates_lockers/closets/secure/security.dm]: should put it in the locker on the maps that have it instead

**Comment by KieranCat** [comment]
Watch out, jughu. Fruitman might lose his mind.

**Comment by ShizCalev** [comment]
Changelog please.

**Comment by ShizCalev** [comment]
image
confirmed for working on monkeys.

**Pull request merged**